coupling_components/mappers/axisymmetric_2d_to_3d.py

```diff
@@ -22,11 +22,13 @@ def __init__(self, parameters):
         self.dir_a = dirs.index(self.settings['direction_axial'])
         self.dir_r = dirs.index(self.settings['direction_radial'])
         self.dir_3d = ({0, 1, 2} - {self.dir_a, self.dir_r}).pop()
+        self.angle  = self.settings.get('angle',360) #angle is set in degrees
```

Comment:
- The formatting in the files is a lot better, but there are still improvements possible. Using the `CTRL` + `ALT` + `L` automatically correctly formats your file. (applies to all files)